docs/pages/desktop-access/getting-started.mdx

```diff
@@ -70,7 +72,8 @@ privileges. You can use this to automate installation, uninstallation, and
 certificate updates. For example:
 
 ```code
-$ teleport-windows-auth-setup-v12.0.0-amd64.exe install --cert=teleport.cer -r
+# Installs Teleport Certificate and reboots
```

Comment:
It does more than install the certificate. It loads a DLL and disable NLA.

Reply:
Updated